conf/messages.en

```diff
@@ -65,12 +65,19 @@ error.number = Please enter a valid number
 error.required = Please enter a value
 error.summary.title = There is a problem
 
-journeyRecovery.continue.title = Sorry, there is a problem with the service
-journeyRecovery.continue.heading = Sorry, there is a problem with the service
-journeyRecovery.continue.guidance = [Add content to explain how to proceed.]
-journeyRecovery.startAgain.title = Sorry, there is a problem with the service
-journeyRecovery.startAgain.heading = Sorry, there is a problem with the service
-journeyRecovery.startAgain.guidance = [Add content to explain why the user needs to start again.]
+journeyRecovery.title = There is a problem
+journeyRecovery.heading = There is a problem
+journeyRecovery.p1 = You have not added any information to this report of receipt yet.
+journeyRecovery.p2 = To submit a report of receipt you must:
+journeyRecovery.numbered1.preLink = Go to the page
```

Comment:
This test should say `Go to the` so it match's the copydeck

Reply:
Ah yeah, good spot. Updating now 👀